flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/CurrentJobsOverviewHandler.java

```diff
@@ -92,24 +101,26 @@ public CurrentJobsOverviewHandler(
 					StringWriter writer = new StringWriter();
 					try {
 						JsonGenerator gen = JsonFactory.JACKSON_FACTORY.createGenerator(writer);
+						final JobDetails.JobDetailsSerializer jobDetailsSerializer = new JobDetails.JobDetailsSerializer();
+
 						gen.writeStartObject();
 
 						if (includeRunningJobs && includeFinishedJobs) {
 							gen.writeArrayFieldStart("running");
-							for (JobDetails detail : result.getRunningJobs()) {
-								writeJobDetailOverviewAsJson(detail, gen, now);
+							for (JobDetails detail : result.getRunning()) {
+								jobDetailsSerializer.serialize(detail, gen, null);
 							}
 							gen.writeEndArray();
 
 							gen.writeArrayFieldStart("finished");
```

Comment:
"finished" --> can use `MultipleJobsDetails.FIELD_NAME_JOBS_FINISHED`

Reply:
Will change it.